docs/colors.md

```diff
@@ -31,6 +31,18 @@ React Native supports also colors as an `int` values (in RGB color mode):
 
 > **_Note:_** This might appear similar to the Android [Color](https://developer.android.com/reference/android/graphics/Color) ints representation but on Android values are stored in SRGB color mode (0xaarrggbb).
 
+## DynamicColorIOS
+
+```jsx
+static DynamicColorIOS({light: [Object object], dark: [Object object]}): [Object object]
+```
+
+The `DynamicColorIOS` function is a platform color type specific to iOS. `DynamicColorIOS` takes a single argument as an object with two keys: light and dark. The values of light and dark themselves must be values returned by the `PlatformColor` function.
```

Comment:
I'm not sure it is true that you have to use `PlatformColor`: https://github.com/facebook/react-native/blob/master/RNTester/js/examples/PlatformColor/PlatformColorExample.js#L239

I think you can specify any color type listed in this file in the `light` and `dark` key: hex, named, rgb, `PlatformColor`, etc. You could probably pass a `DynamicColorIOS` even, but I have no idea what that would do. 😅

Reply:
You're right: they can be any of the traditional string values or an object returned by PlatformColor.   Sorry Kiki, I gave you wrong information offline.    You *cannot* pass a DynamicColorIOS object as the light or dark value to another DynamicColorIOS -- that is enforced at runtime and by the ESLint rule.   Maybe the language can just end after "... with two keys: light and dark."   If we want to be pedantic we could say they values can be a string per https://reactnative.dev/docs/colors or values returned by PlatformColor.

Reply:
removed the comment about platformcolor being the only thing it can take - and removed the static before the dynamiccolorios code block at the top.